c2corg_api/views/user.py

```diff
@@ -187,11 +187,11 @@ def post(self):
                 r = client.redirect_without_nonce(user)
                 response['redirect_internal'] = r
             except:
-                # Any error with discourse must prevent login and validation
+                # Since only the password is changed, any error with discourse
+                # must not prevent login and validation.
```

Comment:
Ok, I see. This is changing the password as part of the "requesting a new password" procedure.